components/application-broker/internal/broker/provision_test.go

```diff
@@ -558,9 +668,23 @@ func TestDoKnativeResourceProvision(t *testing.T) {
 	}
 
 	for _, tc := range []testCase{
+		{
+			name:                           "provision success with new eventing flow",
+			givenCanProvisionOutput:        access.CanProvisionOutput{Allowed: true},
+			newEventingFlow:                true,
+			expectedOpState:                internal.OperationStateSucceeded,
+			expectedOpDesc:                 "provisioning succeeded",
+			expectedEventActivationCreated: true,
+			expectedInstanceState:          internal.InstanceStateSucceeded,
+			initialObjs:                    []runtime.Object{},
+			expectCreates: []runtime.Object{
+				bt.NewIstioPeerAuthentication(string(appNs), fmt.Sprintf("%s%s", appNs, peerAuthNameSuffix)),
```

Comment:
I think with the new eventing flow, the `PeerAuthentication` should not be created.